src/backend/cuda/cudalt.sh

```diff
@@ -32,11 +32,12 @@ if test ! -d "$PIC_DIR" ; then
     mkdir -p "$PIC_DIR"
 fi
 
-CMD="${@:2} -Xcompiler -fPIC -o $PIC_FILEPATH"
+shift
```

Comment:
Could you add a comment on what is being shifted? Maybe it is easier to read if we do the shift right after line 15?

Reply:
Sure, can do.